### ISSUE TITLE
Bug: option deprecation check in SimulationTimeSeries fails if options is not defined

### ISSUE THREAD

**Issue by asnyv** [equinor/webviz-subsurface#968]
This (currently unreleased) check requires that `options` is an `Iterable`, but the default  value is `None`
https://github.com/equinor/webviz-subsurface/blob/4f7d31d8c95cfda9dd6fd2713bb16404a1d3e4e4/webviz_subsurface/plugins/_simulation_time_series/_plugin.py#L51-L61

Suggestion: Either skip if `options is None` or change default for options to an empty list `[]`

**Comment by asnyv** [comment]
@jorgenherje Which of the two suggested solutions do you think would be the most robust?

**Comment by jorgenherje** [comment]
Would early return with `!options` cover both cases - i.e. Both empty list of options or if options is not defined?

**Comment by jorgenherje** [comment]
Yes, that could be. I just kept the type from the old plugin - which stated `dict`. But if `list` is fully representable for the `options` from config `.yaml`-file, it could be a more clean solution.

**Comment by asnyv** [comment]
ah sry, it is a `dict` of course 😂 an empty `dict` as default instead of `None` then? 😉 I just read `iterable` in the error message and my head went straigth to `list` 😅

**Comment by jorgenherje** [comment]
Yes, that could of course be an even better solution! I vote for an empty `dict` as default👍

**Comment by asnyv** [comment]
I'll fix 😊
